packages/lit-dev-content/site/docs/templates/expressions.md

```diff
@@ -333,15 +333,45 @@ Element expressions can occur inside the opening tag after the tag name:
 <div ${ref(elementReference)}></div>
 ```
 
-Expressions **_cannot_** appear where tag or attribute names would appear; however [static expressions](#static-expressions) can.
+### Invalid locations { #invalid-locations }
 
-```html
-<!-- ERROR -->
-<${tagName}></${tagName}>
+Expressions should not appear in the following locations and will issue warnings/errors in development mode:
 
-<!-- ERROR -->
-<div ${attrName}=true></div>
-```
+* Where tag or attribute names would appear. Lit does not support dynamically changing values in this position.
+
+  ```html
+  <!-- ERROR -->
+  <${tagName}></${tagName}>
+
+  <!-- ERROR -->
+  <div ${attrName}=true></div>
+  ```
```

Comment:
Link to static docs

Reply:
The static reference is at the end of the list.